src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -143,11 +172,25 @@ impl KvmVcpu {
         };
         get_all_registers(&self.fd, &mut state.regs).map_err(KvmVcpuError::SaveState)?;
         state.mpidr = get_mpidr(&self.fd).map_err(KvmVcpuError::SaveState)?;
+        state.kvi = self.kvi;
         Ok(state)
     }
 
     /// Use provided state to populate KVM internal state.
-    pub fn restore_state(&self, state: &VcpuState) -> Result<(), KvmVcpuError> {
+    pub fn restore_state(
+        &mut self,
+        vm_fd: &VmFd,
+        state: &VcpuState,
+        index: u8,
```

Comment:
Why `restore_state()` needs to get `index` as an argument. Apparently, `KvmVcpu` has the index as a member variable.

Reply:
fixed